docs/templates/tutorials/gps-post-processing.mdx

```diff
@@ -8,35 +8,37 @@ This tutorial requires:
 * (Optional) For processing improvement: precise ephemeris and clock files from the IGS
 * RTKLIB
 
-Download the latest version of Emlid RTKLIB QT apps for your system: 
+Download the latest version of Emlid RTKLIB QT apps for your system:
 
 <center>
 
 | Windows | Linux |
 |:------------:|:--------------:|
 | [Download](https://files.emlid.com/RTKLIB/rtklib-qt-win-b33.zip)   | [Download](https://files.emlid.com/RTKLIB/rtklib-qt-linux-b33.zip)     |
-| [md5](https://files.emlid.com/RTKLIB/RTKLIB-MD5SUMS)        | [md5](https://files.emlid.com/RTKLIB/RTKLIB-MD5SUMS)          | 
+| [md5](https://files.emlid.com/RTKLIB/RTKLIB-MD5SUMS)        | [md5](https://files.emlid.com/RTKLIB/RTKLIB-MD5SUMS)          |
 
 </center>
 
-Rover track is calculated relatively to the base station so in order to get rover track with correct absolute coordinates the exact position of the base station should be known. You either need to place base station on a point with known coordinate or determine it by post-processing base against a reference station in static mode. It is better if the station is in the range of 100 km (60 miles) for RS2 and 30 km (18 miles) for RS+, but longer range might work as well.
+Rover track is calculated relatively to the base station so in order to get rover track with correct absolute coordinates the exact position of the base station should be known. You either need to place base station on a point with known coordinate or determine it by post-processing base against a reference station in static mode.
+
+It is better if the station is within 100 km range, but longer range might work as well.
```

Comment:
Why did you delete the info about the baseline for Reach RS+? We don't recommend post-processing the L1-logs with the base station logs from 100 km away.